s3tests_boto3/functional/test_s3.py

```diff
@@ -14941,6 +14941,15 @@ def _verify_records(records, bucket_name, event_type, src_keys, record_type, exp
         return len(keys_found) == expected_count and len(keys_found) == len(all_keys)
     return len(keys_found) == expected_count
 
+def _verify_record_field(records, bucket_name, event_type, object_key, record_type, field_name, expected_value):
+    for record in iter(records.splitlines()):
+        parsed_record = _parse_log_record(record, record_type)
+        logger.info('bucket log record: %s', json.dumps(parsed_record, indent=4))
+        if bucket_name in record and event_type in record and object_key in record:
+            if field_name in parsed_record:
+                val = parsed_record[field_name]
```

Comment:
you should probably wrap that with try/except of `KeyError` in case that the field name does not exist

Reply:
Done.